docs/content/latest/tools/superset.md

```diff
@@ -0,0 +1,53 @@
+headerTitle: Apache Superset
+linkTitle: Apache Superset
+description: Use Apache Superset to explore and visulize data in YugabyteDB.
+menu:
+  latest:
+    identifier: superset
+    parent: tools
+    weight: 2794
+isTocNested: true
+showAsideToc: true
+---
+
+[Apache Superset](https://superset.apache.org/) is a fast, lightweight and intuitive open-source data exploration and visualization tool that helps you query your data stored in YugabyteDB and visualize it using simple line charts to highly detailed geospatial charts.
+
+![Superset Dashboard](/images/develop/tools/superset/dashboard.png)
+
+## Before you begin
+
+Your YugabyteDB cluster should be up and running. If you're new to YugabyteDB, create a local cluster in less than five minutes following the steps in [Quick Start](../../quick-start/install). You can also get started with the free tier of [YugabyteDB Fully-Managed Cloud](https://www.yugabyte.com/cloud/). You also need to load a relevant database in your YugabyteDB for exploration and visualizaation. If using yugabyted, you can load the Northwind sample database with the ./bin/yugabyted demo connect command from your shell or you can follow [instructions here](https://docs.yugabyte.com/latest/sample-data/northwind/).
+
+You also need to install Apache Superset to explore and visualize your data. You can install Superset using [Docker Compose](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose) (recommended) or from scratch using [python install (PIP)](https://superset.apache.org/docs/installation/installing-superset-from-scratch).
+
+## Connecting Apache Superset with YugabyteDB
+
+After successful installation, launch Superset in the browser at http://<hostname/IP>:8088. In case of local installation on your laptop, navigate to localhost:8088 or 127.0.0.1:8088. Superset comes with standard PostgreSQL driver that also connects to YugabyteDB. You can also manually install [psycopg2 driver](https://www.psycopg.org/docs/) to connect to YugabyteDB.
+
+Follow these steps to connect Apache Superset to YugabyteDB:
+
+1. Navigate to Data → Databases → + Databases and choose "PostgreSQL" from the “Connect a database” menu as shown below.
+
+![Connect Database](/images/develop/tools/superset/connect-database.png)
+
+2. Enter the hostname or IP Address of your YB-TServer with standard credentials and click "Finish".
+
+![Database Connection Credentials](/images/develop/tools/superset/connect-ybdb.png)
+
+{{< note title="Note" >}}
+As of Docker version 18.03, the host.docker.internal hostname connects to your Docker host from inside a Docker container.
+{{< /note >}}
+
+3. You should now be connected to the YugabyteDB and to verify you can check the availabe databases and schemas under "Data". Navigate to Data → Datasets and click "+Datasets" and in the dialog configure as shown below. Dropdown selectors would show the datasbes and schemas available for explore and visualize operations.
```

Comment:
Number everything with `1.` — it makes editing lists a lot easier later on!

```suggestion
1. Enter your YugabyteDB tablet server's hostname or IP address with standard credentials and click "Finish".

    ![Database Connection Credentials](/images/develop/tools/superset/connect-ybdb.png)

    {{< note title="Note" >}}
As of Docker version 18.03, the `host.docker.internal` hostname connects to your Docker host from inside a Docker container.
    {{< /note >}}

1. Verify that you can access the available databases and schemas under "Data". Navigate to Data > Datasets, and click "+Datasets".

    The dropdown list should show the databases and schemas available to explore and visualize.
```

Reply:
(Also, the weird formatting of the note is correct. If you indent the _content_, it'll render as a code block!)